### PR TITLE
Add documentation for yarn.

### PR DESCRIPTION
**Pull request by schmidtk**
See [README.md](https://github.com/schmidtk/opensphere/blob/yarn-docs/README.md) and [DEVELOPING.md](https://github.com/schmidtk/opensphere/blob/yarn-docs/DEVELOPING.md) for rendered markdown with these changes.

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -9,19 +9,26 @@ Start by [forking](https://github.com/ngageoint/opensphere/fork) the OpenSphere
...
-[Clone|https://help.github.com/articles/cloning-a-repository/] your repository, and change directory into the clone.
+### Workspace Setup
 
-Install the dependencies by running
+#### Yarn
 
-    $ npm install
+Clone [opensphere-yarn-workspace](https://github.com/ngageoint/opensphere-yarn-workspace), and change directory into `opensphere-yarn-workspace/workspace`.
+
+Clone your fork, then run `yarn install`.
```
Comment (wallw-bits) [inline DEVELOPING.md]: I was under the impression this needed to be done in the workspace root. If so, perhaps make that more clear?

File: README.md [hunk 2]
```diff
@@ -19,8 +18,21 @@ what we've done with Openlayers and Cesium but want your own UI? You can do that
...
+The build will not run natively in Windows. Perhaps try Cygwin or Linux via a VM or Docker container. It should run great on OS X and typical Linux distributions.
+
+### Yarn
+
+[Yarn](https://yarnpkg.com/en/) is recommended for dependency management over npm, particularly when using plugins and configuration projects. It will automatically link related projects and dependencies in your workspace.
+
+* Clone [opensphere-yarn-workspace](https://github.com/ngageoint/opensphere-yarn-workspace)
+* Clone OpenSphere to the `workspace` directory
+* `yarn install`
+* `npm run build`
```
Comment (wallw-bits) [inline README.md]: If we are recommending `yarn`, then why not use it to run scripts as well? `yarn run build`

File: DEVELOPING.md (hunk 1 above)
Comment (schmidtk) [reply DEVELOPING.md]: That was my initial assumption, but I later realized yarn will walk up the directory structure, find the workspace root, and run install for the workspace. It doesn't matter where you run it from.

File: README.md (hunk 2 above)
Comment (schmidtk) [reply README.md]: Either works for me. We're assuming npm is installed and those two commands are a horse apiece, so you can run either one. All of our scripts use `npm run`, so I went with that.

**Pull request merged**